libs/pages/onboarding/ui/src/lib/components/step-company/step-company.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, ButtonSize, ButtonType, InputSelect, InputText } from '@console/shared/ui'
```

Comment:
MIssing the form logic of the store connexion to be fully functional but ok !

Does not belong in `ui` either